packages/bpk-component-navigation-stack/readme.md

```diff
@@ -0,0 +1,48 @@
+# bpk-component-navigation-stack
+
+> Backpack navigation stack component.
+
+## Installation
+
+```sh
+npm install bpk-component-navigation-stack --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkNavigationStack from 'bpk-component-navigation-stack';
+
+export default () => (
+  <BpkNavigationStack views={[<div key="1" />]} />
```

Comment:
It was triggering a waring during tests. As far as I know any array of elements needs keys on React. This can be dynamic and react will use the key to decide if it should remount the item in the DOM or just update it.

Reply:
`React.Children.toArray` can be used (see https://reactjs.org/docs/react-api.html#reactchildrentoarray).

I don't think we should force consumers to add keys to the views they are passing.

Reply:
Not sure why the warning is not being triggered in the storybook examples too.

Reply:
I think `Children.toArray` is supposed to be used to flatten the child structure into an array not to add keys to an array (it adds but that's a consequence, not the objective). I'll remove the key from the docs though, it doesn't really seem to trigger a warning in the stories, only in the tests.